testConfig/configData.js

```diff
@@ -0,0 +1,28 @@
+const configData = {
```

Comment:
re: `testConfig/configData.js` since loading of config file is not a part of `no-config` functionality and because test data is small, I sugget to use it as object directly in `test.js` and remove `testConfig/configData.js`. Also I'm sure there would be a need for more test data (separate configs) and see no need to separate test configs from the test code and put them inside multiple files.